content/container_insights/setupalarm.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Setup CloudWatch Alarm"
+chapter: false
+weight: 3
+---
+#### Setup CloudWatch Alarm
+
+In this section, we will setup a CloudWatch alarm and send notification using SNS the number of Pod restarts goes greater than 1 in a 5 minute period.
```

Comment:
we will setup CloudWatch alarm and send notification using SNS when a Pod restarts more than once in a 5 minute period.